src/pool/srv_pool_map.c

```diff
@@ -177,20 +177,30 @@ update_one_tgt(struct pool_map *map, struct pool_target *target,
 			D_INFO("Skip EXCLUDE_OUT DOWNOUT "DF_TARGET"\n",
 			       DP_TARGET(target));
 			break;
-		case PO_COMP_ST_UP:
 		case PO_COMP_ST_UPIN:
-		case PO_COMP_ST_NEW:
 			D_ERROR("Can't EXCLUDE_OUT non-down "DF_TARGET"\n",
 				DP_TARGET(target));
 			return -DER_INVAL;
+		case PO_COMP_ST_NEW:
```

Comment:
here the code is different with master barnch that cannot POOL_EXCLUDE_OUT NEW comp, is it intended to make this difference for 2.2?

Reply:
yes, 2.4 change the NEW/UP target status for extending.